jams.js

```diff
@@ -41,6 +41,9 @@ const _jams =ast=> {
             return _jams(ast.children[0])
         }
         case 'str': {
+            if (ast.text === '""'){
```

Comment:
I actually think checking if the children are none makes more sense than checking for the string. It keeps us relying on the grammar more than this intermediate kind of parsing.

Reply:
so this `ast.children.length == 0` ?

Reply:
looks better to me to, in the end should be the same result